jaxrs/WFLY-10636_EAP7-1010.adoc

```diff
@@ -0,0 +1,74 @@
+= WFLY-10636 / EAP7-1010 RESTEasy extended async / reactive support
```

Comment:
Is there an agreed convention around this?

Reply:
No I don't know about any. I just thought it will be more visible. But important is that it is stated in proposal, you can leave it as is if you like.